docs/source/en/perf_train_gpu_many.md

```diff
@@ -15,143 +15,150 @@ rendered properly in your Markdown viewer.
+solution, and the optimal settings depend on the specific hardware configuration you are using. 
+
+This guide offers an in-depth overview of individual types of parallelism, as well as guidance on ways to combine   
+techniques and choosing an appropriate approach. For step-by-step tutorials on distributed training, please refer to
+the [🤗 Accelerate documentation](https://huggingface.co/docs/accelerate/index). 
 
 <Tip>
 
- Note: Most of the strategies introduced in the [single GPU section](perf_train_gpu_one) (such as mixed precision training or gradient accumulation) are generic and apply to training models in general so make sure to have a look at it before diving into the following sections such as multi-GPU or CPU training.
+While the main concepts discussed in this guide are likely applicable across frameworks, here we focus on 
+PyTorch-based implementations.
 
 </Tip>
 
-We will first discuss in depth various 1D parallelism techniques and their pros and cons and then look at how they can be combined into 2D and 3D parallelism to enable an even faster training and to support even bigger models. Various other powerful alternative approaches will be presented.
-
-## Concepts
+Before diving deeper into the specifics of each technique, let's go over the rough decision process when training 
+large models on a large infrastructure.
 
-The following is the brief description of the main concepts that will be described later in depth in this document.
+## Scalability strategy
 
-1. **DataParallel (DP)** - the same setup is replicated multiple times, and each being fed a slice of the data. The processing is done in parallel and all setups are synchronized at the end of each training step.
-2. **TensorParallel (TP)** - each tensor is split up into multiple chunks, so instead of having the whole tensor reside on a single gpu, each shard of the tensor resides on its designated gpu. During processing each shard gets processed separately and in parallel on different GPUs and the results are synced at the end of the step. This is what one may call horizontal parallelism, as the splitting happens on horizontal level.
-3. **PipelineParallel (PP)** - the model is split up vertically (layer-level) across multiple GPUs, so that only one or several layers of the model are places on a single gpu. Each gpu processes in parallel different stages of the pipeline and working on a small chunk of the batch.
-4. **Zero Redundancy Optimizer (ZeRO)** - Also performs sharding of the tensors somewhat similar to TP, except the whole tensor gets reconstructed in time for a forward or backward computation, therefore the model doesn't need to be modified. It also supports various offloading techniques to compensate for limited GPU memory.
-5. **Sharded DDP** - is another name for the foundational ZeRO concept as used by various other implementations of ZeRO.
+**Parallelization strategy for a single Node / multi-GPU setup**
```

Comment:
I think we can link to[ the space that computes ](https://huggingface.co/spaces/hf-accelerate/model-memory-usage)the RAM needed for each model here as well!

Reply:
Great idea! Added.